pydoctor/test/test_astbuilder.py

```diff
@@ -500,8 +500,7 @@ class C:
         pass
     ''', modname='mod', systemcls=systemcls)
     C = mod.contents['C']
-    assert C.decorators == [(('cd', 'mod.cd', mod.contents['cd']), None)], \
-      C.decorators
+    assert C.decorators == [('mod.cd', None)]
```

Comment:
It looks like this case was also using the custom assert failure message.

It's ok to remove it, and we can added if if needed... not sure if it's still needed.

```suggestion
    assert C.decorators == [('mod.cd', None)], C.decorators
```

Reply:
I think that in general, pytest's reporting on asserts is good enough. Only in special cases, like where we had to ignore the length limit, do we need a custom message.